app/views/users/posts.html.erb

```diff
@@ -0,0 +1 @@
+<h1>Here is a list of posts for User ID: <%= @user_id %></h1>
```

Comment:
- [ ] This file should not be under the `views/users` folder. It should be under the `views/posts` folder, and the file name should be `views/posts/index.html.erb` to match the posts index action.